lib/processors/manifestCreator.js

```diff
@@ -457,24 +488,28 @@ async function createManifest(libraryResource, libBundle, descriptorVersion, _in
 			// - from library resources (if "messagebundle.properties" exists)
 			function i18n() {
 				const i18nElement = findChild(libraryAppData, "i18n");
+				let i18n = null;
 				if ( i18nElement ) {
-					const i18n = i18nElement._;
+					i18n = i18nElement._; // text content
 					if ( i18n === "false" ) {
 						return false;
 					} else if ( i18n === "true" ) {
-						return "messagebundle.properties";
-					} else {
-						return i18n;
+						i18n = "messagebundle.properties";
 					}
 					// log.verbose("  sap.ui5/library/i18n property taken from .library appData: '%s'", library.i18n);
 				} else {
 					if ( libBundle.findResource("messagebundle.properties") != null ) {
 						// log.verbose("  sap.ui5/library/i18n property determined from resources: '%s'", library.i18n);
-						return "messagebundle.properties";
+						i18n = "messagebundle.properties";
 					} else {
 						return false;
 					}
 				}
+				// i18n can be an empty string therefore check for not being null
```

Comment:
But for an empty string `createI18nSection` returns `false` anyways. Is this correct?

Reply:
tackled with: 8eaf458